src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -183,20 +190,20 @@ describe('NameAndPictureScreen', () => {
     expect(getByText('nameAndPicGuideCopyContent')).toBeTruthy()
   })
   it('does not render skip button when configured so', () => {
-    // TODO replace route param with mock Statsig flag
+    mockStatsigGet = jest.fn().mockReturnValue(false)
```

Comment:
I'm very surprised this works, but I still don't think we should do it. see here: https://github.com/valora-inc/wallet/pull/2951